CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Master
 
+##### Enhancements
+
+* Modified KeyboardToolbar to keep track of the next, back, and done buttons, so they can be enabled / disabled as needed. 
```

Comment:
Backticks can be used. Also I don't think a comma is needed after buttons.
```
* Modified `KeyboardToolbar` to keep track of the next, back, and done buttons so they can be enabled / disabled as needed.
```

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/f0fb818e19adc3fe77eecb915f01f9ef479cc927